manpages/awesome.1.txt

```diff
@@ -46,6 +46,8 @@ OPTIONS
     Add a directory to the library search path.
 *-a*, *--no-argb*::
     Don't use ARGB visuals.
+*-m*, *--screen*:: 'manual' or 'auto'::
+    Use "manual" to execute rc.lua before creating the screens.
```

Comment:
Should explain maybe also what you need to do then (the requirement of creating it manually).  Also with the error message for it then (if that's not the case already).

Reply:
Technically there's nothing to do. However, if you modified your config and it uses `mouse.screen` in the global context (aka, called directly from `rc.lua` rather than a signal callback), then there will be some errors.